client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx

```diff
@@ -43,11 +43,12 @@ const AddReviewDialog = ({ productId }) => {
       .catch(err => console.log(err));
   }
 
-  const classes = useStyles();
+  // const classes = useStyles();
 
   return (
-    <div className={classes.buttons}>
-      <Button variant='contained' color='primary'>MORE REVIEWS</Button>
+    // <div className={classes.buttons}>
+          <div>
+      {/* <Button variant='contained' color='primary'>MORE REVIEWS</Button> */}
```

Comment:
Consider deleting all the comments in this file.

Reply:
Thank you, removing